### PR TITLE
Fix minor bugs with disconnected agent deployments

### PR DESCRIPTION
**Pull request by zaneb**

### CODE REVIEW

**Review by pawanpinjarkar — COMMENTED** [review]
/lgtm

**Review by rwsu — COMMENTED** [review]
/lgtm

File: pkg/asset/agent/image/ignition.go [hunk 1]
```diff
@@ -163,14 +163,27 @@ func (a *Ignition) Generate(dependencies asset.Parents) error {
 		return err
 	}
 
-	// Use bootstrap script to get container images
-	err = bootstrap.AddStorageFiles(&config, "/usr/local/bin/release-image.sh",
-		"bootstrap/files/usr/local/bin/release-image.sh.template",
-		struct{ ReleaseImage string }{ReleaseImage: agentManifests.ClusterImageSet.Spec.ReleaseImage})
-	if err != nil {
+	// Set up bootstrap service recording
```
Comment (rwsu) [inline pkg/asset/agent/image/ignition.go:166]: I was trying to understand why bootstrap-service-record.sh is being pulled in to the ignition. I see that release-image-download.sh uses it.

**Pull request merged**